src/scripts/main.ts

```diff
@@ -1,36 +1,37 @@
+import {
+    GitHubTeams,
+    GitHubMembers,
+    GitHubRepos
+} from "@co-digital/api-sdk/lib/api-sdk/github/type";
+
 import { saveToFile } from "../utils/fs";
+import { OrgData } from "../types/config";
+import { getData } from "../service/github";
 import {
+    GIT_BASE_URL,
+    MEMBERS_KEY,
     REPOS_FILE_PATH,
-    ORG_DATA
+    REPOS_KEY,
+    TEAMS_KEY
 } from "../config/index";
 import {
-    getPerTeamData,
-    setPerTeamData,
-    setOrgData,
     setTeamsData,
     setMembersData,
     setReposData,
+    setTeamsMembersReposInnerData
 } from "../utils/index";
-import { getTeamsData, getMembersData, getReposData } from "../service/github";
 
 export const main = async (org: string): Promise<void> => {
     try {
+        const data = { } as OrgData;
```

Comment:
I would give this a more semantic name to match the interface `const orgData = { } as OrgData`.